.travis.yml

```diff
@@ -26,6 +26,53 @@ jobs:
       env:
         - ROS_DISTRO=noetic
         - PYTHON_SUFFIX="3.7"
+      addons:
```

Comment:
All those additions in the travis file are probably in by accident. We already moved them to the github actions